cmd/genji/shell/command.go

```diff
@@ -53,6 +53,15 @@ func runTablesCmd(db *genji.DB, cmd []string) error {
 // displayTableIndex prints all indexes that the given table contains.
 func displayTableIndex(db *genji.DB, tableName string) error {
 	ctx := context.Background()
+	_, err := db.QueryDocument(ctx, "SELECT table_name FROM __genji_tables WHERE table_name = ?", tableName)
+	switch err {
+	case database.ErrDocumentNotFound:
+		return database.ErrTableNotFound
+	case nil:
+		// nop
+	default:
+		return err
+	}
 
 	res, err := db.Query(ctx, "SELECT * FROM __genji_indexes WHERE table_name = ?", tableName)
```

Comment:
Both queries should be in the same transaction (using `db.View`).

Reply:
thank you @tie